cypress/test/campaignTests/AnnouncementHero.cy.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import AnnouncementHero from '../../../components/campaigns/AnnoucementHero';
+
+describe('AnnouncementHero Component', () => {
+  it('should render the component when the date is within the valid range', () => {
+    // Set the current date to May
+    const mockDate = new Date('2023-05-01T00:00:00Z');
+    cy.clock(mockDate.getTime());
+
+    mount(<AnnouncementHero />);
+
+    // Assert that the component is rendered
+    cy.get('.bg-gray-50').should('exist');
```

Comment:
We can use, `data-test-id`, in the original component, and use it in cypress for more accurate testing? 
cc: @akshatnema @reachaadrika

Reply:
I think @reachaadrika is using it at many places. Maybe she forgot about it here, but yeah, the test for background color should be there as it tests whether component is rendering it's properties also.

Reply:
I am checking for background and for rest of the components I will add data attribute

Reply:
done the suggested changes , do check @imabp

Reply:
done